.github/workflows/build.yml

```diff
@@ -0,0 +1,98 @@
+name: Build
+
+on: [push]
+
+jobs:
+  Linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y build-essential cmake libgl-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev
```

Comment:
```suggestion
          sudo apt-get install -y build-essential cmake libgl-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev ninja-build
```

Reply:
What does using ninja add here?

Reply:
Could you perhaps make an PR to my branch? I've not used ninja before. I can speed up the build by parallelizing the build process as well. I'll have to check how many cores the github runners have.

Reply:
I decided against using Ninja and using the `--parallel` option instead because Ninja did not offer a speed benefit.

Reply:
While they might on par now, `make` still is inferior in muxing the logs from all its worker threads, unlike ninja. See this part from your run https://github.com/sharkwouter/OpenNFS/actions/runs/11840413929/job/32994177926#step:4:377:

![image](https://github.com/user-attachments/assets/16a3f42c-1087-4db1-802a-8c1cb0afd7bf)

This should still move to ninja to provide user with clear information about where the messages originate from.